Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/Update/UpdateStreetcodeHandler.cs

```diff
@@ -230,6 +237,111 @@ private void UpdateAudio(IEnumerable<AudioUpdateDTO> audios, StreetcodeContent s
             }
         }
 
+        private async Task UpdateArtGallery(StreetcodeContent streetcode, IEnumerable<StreetcodeArtSlideCreateUpdateDTO> artSlides, IEnumerable<ArtCreateUpdateDTO> arts)
+        {
+            _repositoryWrapper.StreetcodeArtRepository.DeleteRange(await _repositoryWrapper.StreetcodeArtRepository.GetAllAsync(a => a.StreetcodeId == streetcode.Id));
+
+            var toCreateArts = new List<Art>();
```

Comment:
In this case, if it too hard to configure with mapper then I would suggest to make it a separate private function to improve code readability